diff --git a/src/Npgsql/PGUtil.cs b/src/Npgsql/PGUtil.cs
index 88ac203..5a3cbae 100644
--- a/src/Npgsql/PGUtil.cs
+++ b/src/Npgsql/PGUtil.cs
@@ -122,7 +122,7 @@ internal static Task<TResult> TaskFromResult<TResult>(TResult result)
         public static Exception ThrowIfReached(string message = null)
         {
             Contract.Requires(false);
-            return message == null ? new Exception("An internal  EnterpriseDB.EDBClient occured, please open an issue to support.enterprisedb.com with this exception's stack trace") : new Exception(message);
+            return message == null ? new Exception("An internal  EnterpriseDB.EDBClient occured, please open an issue to support@enterprisedb.com with this exception's stack trace") : new Exception(message);
         }
 
         internal static bool IsWindows =>
